test/specs/configs/full.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018 Adobe. All rights reserved.
+# This file is licensed to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+# OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+
+# defines the default content repository
+# content: local
+content: https://github.com/adobe/project-helix.io/blob/master
```

Comment:
I would prefer to have the default configuration in the `default` strain instead of having them as top-level elements.

Reply:
I wanted to preserve the minimal config to keep @davidnuescheler happy. w/o the toplevel defaults, the minimal config you need for enabling a different content repo would be:

```yaml
strains:
  default:
    content: https://github.com/adobe/project-helix.io/blob/master
```